nlu/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.9'
```

Comment:
Please add these ROOT/docker/docker-compose.nlu.yml and ROOT/docker/docker-compose.nlu.dev.yml